sympy/printing/printer.py

```diff
@@ -4,14 +4,26 @@
 passed to a designated Printer who then is responsible to return an
 adequate representation of that expression.
 
-The basic concept is the following:
+**The basic concept is the following:**
   1. Let the object print itself if it knows how.
   2. Take the best fitting method defined in the printer.
   3. As fall-back use the emptyPrinter method for the printer.
 
-Some more information how the single concepts work and who should use which:
+Which Method is Responsible for Printing?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-1. The object prints itself
+The whole printing process is started by calling ``.doprint(expr)`` on the printer
+which you want to use. This method looks for an appropriate method which can
+print the given expression in the given style that the printer defines.
+While looking for the method, it follows these steps:
+
+1. **Let the object print itself if it knows how.**
+
+    The printers look for a specific method in every object. The name of that method
```

Comment:
"The printers look" -> "The printer looks"?

Reply:
Fixed.